src/store/permissions/reducer.js

```diff
@@ -0,0 +1,39 @@
+import { applyReducerHash } from '@redhat-cloud-services/frontend-components-utilities/files/ReducerRegistry';
+import { ACTION_TYPES } from '../../constants';
+
+const loadWritePermissionsPending = (state) => ({
+    ...state,
+    loadingFailed: false,
+    loading: !insights.chrome.isProd,
```

Comment:
Shouldn't this be always true?

Reply:
This is just a safe check. Pending should not never happen when users are on prod anyway.